### PR TITLE
Add PSTN and 1:N storybook examples and quickstart guides

### PR DESCRIPTION
**Pull request by edwardlee-msft**
# What
Add PSTN and 1:N storybook examples and quickstart guides. Add entries of api additions into storybook Adapter page.

This PR holds quickstart documentation and also files for storybook preview examples of both 1:N and PSTN.

PSTN storybook preview example:

https://user-images.githubusercontent.com/97130533/187005727-ca181608-9da4-44ca-b158-b5634e2a8bd4.mov

1:N storybook preview example:

https://user-images.githubusercontent.com/97130533/187005901-9bb7ed2e-b3ac-449a-b72b-6eb469670c34.mp4

This PR is dependent on https://github.com/Azure/communication-ui-library/pull/2232 to ensure 1:1 calls do not end the call when put on hold and to ensure that videogallery and participant pane is visible when put on hold.

# Why
Provide a way for Contoso to test PSTN and 1:N. For testing 1:N (inbound), contoso would need to call an ACS user that has inbound call setup on their end. The 1:N storybook example only calls the ACS user.

# How Tested
Tested locally on MacOS with Chrome browser.

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/CallComposite/CallCompositeDocs.tsx [hunk 1]
```diff
@@ -193,9 +193,10 @@ export const getDocs: () => JSX.Element = () => {
       <Description>
         As well as these participantIDs you are required to provide a [phone
         number](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/telephony/get-phone-number?tabs=windows&pivots=platform-azcli)
-        from a Azure Communications resource to the composite's adapter for PSTN. This phone number serves as your
-        caller ID when calling PSTN. This phone number is not required for calling other Azure Communications users,
-        however, will be required if you are looking to call a ACS user and a PSTN user in the same call.
+        known as an AlternateCallerId from a Azure Communications resource to the composite's adapter for PSTN. This
```
Comment (anjulgarg) [inline packages/storybook/stories/CallComposite/CallCompositeDocs.tsx:196]: `known as an AlternateCallerId` -> `as an alternate caller id`

Since the statement is misleading. The phone number is phone number in this case and not necessarily "known" as an AlternateCallerId.

**Comment by anjulgarg** [comment]
Screen Shot 2022-08-18 at 10 24 10 AM
Would this URL for for 1:N/PSTN scenarios?
I don't think so.

Possible to remove it?

File: packages/storybook/stories/controlsUtils.ts [hunk 2]
```diff
@@ -161,6 +162,16 @@ export const controlsToAdd = {
     defaultValue: '',
     name: 'Call locator (ACS group ID, Teams meeting link, or Room ID)'
   },
+  callParticipantsLocator: {
+    control: 'array',
+    defaultValue: ['+###########'],
+    name: 'Call locator (Participant IDs)'
```
Comment (dmceachernmsft) [inline packages/storybook/stories/controlsUtils.ts:168]: Should this maybe be participants phoneNumbers?

**Review by dmceachernmsft — APPROVED** [review]

**Comment by dmceachernmsft** [comment]
> Screen Shot 2022-08-18 at 10 24 10 AM
> 
> Would this URL for for 1:N/PSTN scenarios? I don't think so.
> Possible to remove it?

I think we should remove this as well if possible

**Review by carocao-msft — APPROVED** [review]

File: packages/storybook/stories/CallComposite/Quickstart1ToNDocs.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+  title="Composites/CallComposite/1:N Call/1:N Get Started"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Getting Started With 1:N
+
+<DetailedBetaBanner />
+
+The UI Library CallWithChat Composite and Calling Composite supports 1:N by enabling developers to perform outbound calls through
+Azure Communication Services call service. Currently, Inbound calls (receiving calls from outbound ACS calls) is not supported by
```
Comment (dmceachernmsft) [inline packages/storybook/stories/CallComposite/Quickstart1ToNDocs.stories.mdx:20]: I would reword this saying that inbound scenarios aren't supported by our composite level offerings or something since we know we can build a component app that can receive inbound calls.

File: packages/storybook/stories/CallComposite/Quickstart1ToNDocs.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+The following features are supported:
+
+1. 1:1 outbound calls with ACS users
+2. 1:N outbound calls with ACS users
+3. 1:N outbound calls with both ACS and PSTN users if an alternate caller Id is provided.
+4. [On/Off hold features](?path=/docs/composite-adapters--page)
+5. Removing participants from a call
+
+_Unfortunately, users are not able to join into an existing call through a copy invite link for 1:N and PSTN scenarios at this time._
```
Comment (dmceachernmsft) [inline packages/storybook/stories/CallComposite/Quickstart1ToNDocs.stories.mdx:31]: I would re word or remove this since invite links wont be supported for PSTN or 1:N since its a different kind of call than a groupCall.

File: packages/storybook/stories/CallComposite/QuickstartPSTNDocs.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+4. 1 to 1 participant PSTN calls
+5. Multi participant PSTN calls
+6. Dialing/Adding PSTN participants into an existing call
+7. Removing PSTN participants from a call
+
+_Unfortunately, users are not able to join into an existing call through a copy invite link for 1:N and PSTN scenarios at this time._
+
+## Download the Code
+
+You can find the completed code for this quickstart here: [Get started with Composites](https://github.com/Azure-Samples/communication-services-javascript-quickstarts/tree/main/ui-library-quickstart-composites)
```
Comment (dmceachernmsft) [inline packages/storybook/stories/CallComposite/QuickstartPSTNDocs.stories.mdx:37]: hows the updates to these happening? are 1:N and PSTN getting their own samples at some point and this is going to be changed?

File: packages/storybook/stories/CallComposite/components/IncomingCallAlert.tsx [hunk 6]
```diff
@@ -0,0 +1,100 @@
+import { DefaultButton, DefaultPalette, mergeStyles, Persona, PersonaSize, Stack, useTheme } from '@fluentui/react';
+import { Call20Filled, CallEnd20Filled } from '@fluentui/react-icons';
+import React from 'react';
+
+const palette = DefaultPalette;
```
Comment (dmceachernmsft) [inline packages/storybook/stories/CallComposite/components/IncomingCallAlert.tsx:5]: are opposed to exporting this component in the package until we get designs to what this might look like?

File: packages/storybook/stories/CallComposite/QuickstartPSTNDocs.stories.mdx (hunk 5 above)
Comment (edwardlee-msft) [reply packages/storybook/stories/CallComposite/QuickstartPSTNDocs.stories.mdx:37]: We will be using the composite quickstarts and modifying the App.tsx.
I will change the text to reflect that! Good catch!

File: packages/storybook/stories/CallComposite/components/IncomingCallAlert.tsx (hunk 6 above)
Comment (edwardlee-msft) [reply packages/storybook/stories/CallComposite/components/IncomingCallAlert.tsx:5]: Good point, I could add this component into SimpleInboundHomeScreen instead of having a separate component. 
That would resolve any issues with exporting this specific component.
